_posts/2023-08-22-using-plurimath.adoc

```diff
@@ -0,0 +1,282 @@
+
+OMML example processing below will also follow same steps.
+
+```ruby
+  input_string = <<~OMML
+    <m:oMathPara xmlns:m="http://schemas.openxmlformats.org/officeDocument/2006/math" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:mo="http://schemas.microsoft.com/office/mac/office/2008/main" xmlns:mv="urn:schemas-microsoft-com:mac:vml" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main" xmlns:w10="urn:schemas-microsoft-com:office:word" xmlns:w14="http://schemas.microsoft.com/office/word/2010/wordml" xmlns:w15="http://schemas.microsoft.com/office/word/2012/wordml" xmlns:wne="http://schemas.microsoft.com/office/word/2006/wordml" xmlns:wp="http://schemas.openxmlformats.org/drawingml/2006/wordprocessingDrawing" xmlns:wp14="http://schemas.microsoft.com/office/word/2010/wordprocessingDrawing" xmlns:wpc="http://schemas.microsoft.com/office/word/2010/wordprocessingCanvas" xmlns:wpg="http://schemas.microsoft.com/office/word/2010/wordprocessingGroup" xmlns:wpi="http://schemas.microsoft.com/office/word/2010/wordprocessingInk" xmlns:wps="http://schemas.microsoft.com/office/word/2010/wordprocessingShape">
+      <m:oMath>
+        <m:nary>
+          <m:naryPr>
+            <m:chr m:val="∑"/>
+            <m:limLoc m:val="undOvr"/>
+            <m:subHide m:val="0"/>
+            <m:supHide m:val="0"/>
+          </m:naryPr>
+          <m:sub>
+            <m:r>
+              <m:t>i</m:t>
+            </m:r>
+            <m:r>
+              <m:t>=</m:t>
+            </m:r>
+            <m:r>
+              <m:t>1</m:t>
+            </m:r>
+          </m:sub>
+          <m:sup>
+            <m:r>
+              <m:t>n</m:t>
+            </m:r>
+          </m:sup>
+          <m:e>
+            <m:sSup>
+              <m:sSupPr>
+                <m:ctrlPr>
+                  <w:rPr>
+                    <w:rFonts w:ascii="Cambria Math" w:hAnsi="Cambria Math"/>
+                    <w:i/>
+                  </w:rPr>
+                </m:ctrlPr>
+              </m:sSupPr>
+              <m:e>
+                <m:r>
+                  <m:t>i</m:t>
+                </m:r>
+              </m:e>
+              <m:sup>
+                <m:r>
+                  <m:t>3</m:t>
+                </m:r>
+              </m:sup>
+            </m:sSup>
+          </m:e>
+        </m:nary>
+      </m:oMath>
+    </m:oMathPara>
+  OMML
+  formula = Plurimath::Math.parse(input_string, :omml)
+  formula.to_asciimath # >> "sum_(\"i\" = 1)^(\"n\") \"i\"^(3)"
+```
+
+We also support *OMML*, allowing you to work with mathematical content efficiently in *Office* documents.
+
+NOTE: *MathML* string supports *displaystyle* but *OMML* don't have any attribute or tag to understand it, so you will have to pass it explicitly to `to_omml(display_style: (true or false))`
```

Comment:
```suggestion
NOTE: *MathML* string supports *displaystyle* but *OMML* don't have any one attribute or tag corresponding to it. So in order to get the same effect of normal vs compact rendering of equations, you will have to pass the `display_value` argument explicitly to the OOML rendering request, as `to_omml(display_style: (true or false))`
```

Reply:
I will add a separate section for output as well.